src/state/erc20/hooks.ts

```diff
@@ -0,0 +1,82 @@
+import { TokenErc20 } from '@gnosis.pm/dex-js'
+
+import useGlobalState from 'hooks/useGlobalState'
+
+import { Network } from 'types'
+
+import { saveMultipleErc20 } from './actions'
+import { Erc20State } from './types'
+import { buildErc20Key } from '.'
+import { useCallback } from 'react'
+
+export type SingleErc20State = TokenErc20 | null
+
+/**
+ * Syntactic sugar to get erc20 from global state.
+ * Returns a function that fetches the TokenErc20 or null for the given address
+ *
+ * @param networkId The network id
+ */
+export const useGetErc20FromGlobalState = <State extends { erc20s: Erc20State }>(
```

Comment:
I think this method should be called `useErc20` and just get that specific erc20 from the global state. No not returning a callback so someone can call from outside

Reply:
As I mentioned in a comment above, this name is to avoid clashing with the hook that will also fetch the token from the network if needed.

What should I call that one, then?

Reply:
I close this one, to leave the other thread open